implementations/rust/ockam/ockam_app/linux-desktop-template

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
Which kind of templating syntax does this mechanism use? My understanding is it's a feature provided by Tauri, so I'm not challenging it, just wanted to identify it.

As a minor nit, I also would personally find this combination of filename and folder location a little bit unclear about its purpose without having read this PR.

Reply:
> Which kind of templating syntax does this mechanism use? 

Actually, I didn't investigate it, I just added upon the existing template


> As a minor nit, I also would personally find this combination of filename and folder location a little bit unclear about its purpose without having read this PR.

What about `packaging/linux/application-description-template`? Do you have some suggestions?

Reply:
I think your suggestion here is a good improvement 👍🏻